### PR TITLE
feat: UI v3 nav

### LINKED ISSUE #201: Feature: UI updated nav
### Type of feature

🍕 Feature

### Current behavior

Screen Shot 2022-07-15 at 6 22 34 AM

1. Search box needs to move
2. New logo
3. Login is now a button

### Suggested solution

Update nav 
- [ ] Votes link in the header
- [ ] New sign in button
- [ ] New logo and wordmark (separated)

Keep in mind "my votes" will need to be shown only when logged in.

Right side of the nav

Screen Shot 2022-07-15 at 6 32 44 AM

Left side of the nav
Screen Shot 2022-07-15 at 6 32 40 AM

Link to [figma](https://www.figma.com/file/Rrc4Bc5lFoyDOrvD00ThkB/bdougie---new-hot.opensauced?node-id=0%3A1)
 
related https://github.com/open-sauced/hot/issues/200

### Additional context

password to figma is secretsauce

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by shamimbinnur**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
This PR is to update the UI-V3 Nav

## Related Tickets & Documents

UI updated nav #201
resolves #201

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by bdougie** [comment]
@open-sauced/triage can you take a look and QA.

I already notice a few places where links are not working. 

1. Clicking on the logo should return to "/"
2. Clicking on "My Votes" does not filter the results.

Screen Shot 2022-07-18 at 9 23 04 AM

**Review by bdougie — CHANGES_REQUESTED** [review]
All visual changes need a screenshot in this section.

> Mobile & Desktop Screenshots/Recordings

Also left a comment about links not working.

File: tailwind.config.cjs [hunk 1]
```diff
@@ -20,12 +20,16 @@ module.exports = {
         darkGrey: "#171c1d",
         darkestGrey: "#2C3137",
         accent: "#FAEACE",
+        osGray: "#11181C",
```
Comment (bdougie) [inline tailwind.config.cjs:23]:
```suggestion
        osGrey: "#11181C",
```
The spelling works both ways in English, but the project has opted for using "e" in Grey

**Comment by shamimbinnur** [comment]
> @open-sauced/triage can you take a look and QA.
> 
> I already notice a few places where links are not working.
> 
> 1. Clicking on the logo should return to "/"
> 2. Clicking on "My Votes" does not filter the results.
> 
> Screen Shot 2022-07-18 at 9 23 04 AM

@bdougie can you be a little specific about what should I show when someone clicks on votes?

**Comment by bdougie** [comment]
> @bdougie can you be a little specific about what should I show when someone clicks on votes?

Currently in the app, you can vote on projects using the upvote button. Everything the logged in user votes is available to view in this list. 

Screen Shot 2022-07-18 at 11 52 49 AM

File: src/components/GradBackground.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { FC } from 'react'
+import gradBG from '../../public/gradBG.svg'
+
+
```
Comment (NsdHSO) [inline src/components/GradBackground.tsx:4]: If you did ` npm run format` don't you catch these empty lines

Comment (bdougie) [reply src/components/GradBackground.tsx:4]: Hey @NsdHSO you proposed this as a question, but it doesn't needed to be. When reviewing, try to make the feedback actionable. We can discussions in the comments or discord.

File: src/components/GradBackground.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+const GradBackground: FC = ({ children }) => {
+  return (
+    <div className="h-fit relative">
+      <div className="absolute w-[100vw] left-[-3px] z-[-1]">
+        <img className='mx-auto' src={gradBG} alt="" />
```
[the remark this answers is not in the document]
Comment (bdougie) [reply src/components/GradBackground.tsx:9]: Same here. In reviews, try to provide actionable feedback. This definitely needs alt text.

**Comment by bdougie** [comment]
Just noticed that the version is missing from the dropdown.
Screen Shot 2022-07-18 at 3 21 56 PM

**Review by bdougie — APPROVED** [review]
Thanks for working on this.

**Comment by bdougie** [comment]
> Currently in the app, you can vote on projects using the upvote button. Everything the logged in user votes on is available to view in this list.

Working in this in a new PR https://github.com/open-sauced/hot/issues/213

**Review by bdougie — APPROVED** [review]
We really need this new UI shipped this week. I created the issue for the votes section so it can be worked on in a new PR.

**Pull request merged**